### PR TITLE
[MIG] dbfilter_from_header: Migration to 12.0

### PR DESCRIPTION
**Pull request by remytms**
This migrates the module `dbfilter_from_header` to 12.0.

There is a refactor that was proposed to 11.0: #1340.

Should we wait before the refactor is merge into 11.0 or can we migrate this module to 12.0 and than migrate the changes from 11.0 to 12.0 ?

### CODE REVIEW

**Comment by houssine78** [comment]
👍 I've tested it. It works fine on v12

File: dbfilter_from_header/README.rst
Comment (tbaden) [inline dbfilter_from_header/README.rst]: please use the new split readme [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

File: dbfilter_from_header/__manifest__.py
Comment (tbaden) [inline dbfilter_from_header/__manifest__.py]: if you dont use the tag, just delete it.

File: dbfilter_from_header/__manifest__.py
Comment (tbaden) [inline dbfilter_from_header/__manifest__.py]: if you dont use the tag, just delete it.

File: dbfilter_from_header/__manifest__.py
Comment (tbaden) [inline dbfilter_from_header/__manifest__.py]: if you dont use the tag, just delete it.

File: dbfilter_from_header/override.py
Comment (tbaden) [inline dbfilter_from_header/override.py:26]: is it really necessary to log?

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review, just some small changes are needed (see below)

File: dbfilter_from_header/override.py
Comment (houssine78) [reply dbfilter_from_header/override.py:26]: it works this way in the previous versions

**Comment by houssine78** [comment]
@remytms could you have a quick look to make the few requested changes ?

**Comment by remytms** [comment]
Yes, I will fix that as soon as possible. :)

**Comment by remytms** [comment]
@tbaden I have split the readme according to maintainer-tools and fixed empty keys in manifest. :)

File: dbfilter_from_header/__manifest__.py
Comment (tbaden) [inline dbfilter_from_header/__manifest__.py:13]: please add the website tag with the value: https://github.com/OCA/server-tools

**Review by tbaden — CHANGES_REQUESTED** [review]
@remytms thanks for the changes, just something minor.

**Comment by remytms** [comment]
@tbaden website tag added. Thanks for your attentive review.

**Review by tbaden — APPROVED** [review]

**Comment by remytms** [comment]
@houssine78 Can you approve this PR, if it seams good to you ? :)

**Review by houssine78 — APPROVED** [review]

**Pull request merged**
